evals/features/cve/test_cvss_diff.py

```diff
@@ -30,11 +29,6 @@ def __init__(self, cve_id, has_diff):
 def is_cvss_valid(cvss_str: str) -> bool:
     """return True if cvss_str is a valid CVSS3 vector"""
     try:
-        # FIXME: cvss_str is *sometimes* prefixed with the actual score, which
-        # breaks the validation.  Should we canonicalize the output from Aegis
-        # instead?
-        cvss_str = re.sub(r"^[0-9]+(\.[0-9])+/", "", cvss_str)
-
         cvss.cvss3.CVSS3(cvss_str)
```

Comment:
come to think of it we can use this to validate on pydantic model itself for these features